### PR TITLE
Chore: add story layout page

### PR DESCRIPTION
**Pull request by jonniebigodes**
With this pull request the configure section is expanded and the `story-layout` page is introduced. I went with a simple approach.

What was done:
- Created the markdown file for the page.
- Created a snippet to be used as an example.
- Updated the `toc.js` file to include the page.

Feel free to provide feedback

### CODE REVIEW

File: docs/configure/story-layout.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: 'Story layout'
+---
+
+The `layout`[global parameter](../writing-stories/parameters.md) is a way to configure how stories are rendered in Storybook's UI.
```
Comment (domyen) [inline docs/configure/story-layout.md:5]:
```suggestion
The `layout` [global parameter](../writing-stories/parameters.md) allows you to configure how stories are positioned in Storybook's Canvas tab. 
```

File: docs/configure/story-layout.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+<CodeSnippets
+  paths={[
+    'common/storybook-preview-layout-param.js.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+With the example above, Storybook will center all stories in the UI. Asides from `centered` you can also use:
```
Comment (domyen) [inline docs/configure/story-layout.md:19]:
```suggestion
In the example above, Storybook will center all stories in the UI. `layout` accepts these options:
```

File: docs/configure/story-layout.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- `fullscreen`, to emulate a fullscreen environment.
```
Comment (domyen) [inline docs/configure/story-layout.md:21]:
```suggestion
- `centered`: center the component horizontally and vertically in the Canvas
- `fullscreen`: allow the component to expand to the full width and height of the Canvas
```

File: docs/configure/story-layout.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- `padded`, for some extra padding.
```
Comment (domyen) [inline docs/configure/story-layout.md:22]:
```suggestion
- `padded`: Add extra padding around the component
```

File: docs/configure/story-layout.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+If you need to use your own styles, or use a more granular approach we recommend using [decorators](../writing-stories/decorators.md) instead.
```
Comment (domyen) [inline docs/configure/story-layout.md:24]:
```suggestion
If you want to use your own styles, or require a more granular approach we recommend using [decorators](../writing-stories/decorators.md) instead.
```

**Review by domyen — CHANGES_REQUESTED** [review]
thanks, some minor changes

**Comment by jonniebigodes** [comment]
@domyen thank you for the feedback. Going to merge it

**Pull request merged**